src/arr/compiler/anf-loop-compiler.arr

```diff
@@ -101,6 +101,8 @@ j-for = J.j-for
 j-raw-code = J.j-raw-code
 make-label-sequence = J.make-label-sequence
 
+var cases-dispatches = cl-empty
```

Comment:
Not really, because we never iterate over those dictionaries and output everything they contain (the way you have to, with including cases-dispatch in the compiled output).

When you compile test.arr, a single file, that doesn't necessarily mean you needed to recompile anything else.  Try compiling phaseB, and see if things get corrupted.